OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -19,16 +19,20 @@ class ParachutableInfo : ITraitInfo
 		[Desc("If we land on invalid terrain for my actor type should we be killed?")]
 		public readonly bool KilledOnImpassableTerrain = true;
 
+		[Desc("Group where *CorposeSequence is looked up.")]
+		public readonly string CorpseSequenceCollection = "explosion";
+
 		public readonly string GroundImpactSound = null;
-		public readonly string GroundCorpseSequence = "corpse";
+		[SequenceReference("CorpseSequenceCollection")] public readonly string GroundCorpseSequence = "corpse";
 		public readonly string GroundCorpsePalette = "effect";
 
 		public readonly string WaterImpactSound = null;
-		public readonly string WaterCorpseSequence = "small_splash";
+		[SequenceReference("CorpseSequenceCollection")] public readonly string WaterCorpseSequence = null;
```

Comment:
If I leave the value which is ra mod exclusive, the linter will error for every other which does not have the sprite sequences.